python/views/widget_delegate.py

```diff
@@ -114,6 +118,10 @@ def _create_editor_widget(self, model_index, style_options, parent):
         """
         # the default implementation just calls _create_widget for backwards
         # compatibility.
+
+        if not model_index.isValid():
+            return None
```

Comment:
a comment here would be helpful. Why are we exiting early? What's the use case when the index isn't valid?